test/infrastructure/docker/hack/update-deps.sh

```diff
@@ -1,29 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
This was a second way to update deps in capd aside from the `make modules` target